src/pages/login/component/SignUpInput.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import './SignUpInput.scss';
+
+const SignUpInput = ({ id, title, type, name, placeholder }) => {
+  return (
+    <div>
+      <div className="SignUpInput" key={id}>
```

Comment:
```suggestion
      <div className="signUpInput" key={id}>
```
className은 카멜케이스 즉, 소문자로 시작해야 합니다.
대문자로 시작하는 건 컴포넌트 이름입니다!